### PR TITLE
[Ahuna] Add try button to edition menu in Assistant Details

### PR DESCRIPTION
**Pull request by philipperolet**
## Description
The try button below
Screenshot from 2024-02-05 18-55-24

### CODE REVIEW

**Review by flvndvd — COMMENTED** [review]
Can you merge master so we don't see the "delete button" related work?

**Comment by philipperolet** [comment]
thought I had, sorry done

File: front/components/assistant/AssistantDetails.tsx [hunk 1]
```diff
@@ -93,6 +93,9 @@ export function AssistantDetails({
     setIsUpdatingScope(false);
   };
 
+  const showTryButtonInMenu =
+    agentConfiguration.scope === "published" &&
+    agentConfiguration.userListStatus === "not-in-list";
```
Comment (flvndvd) [inline front/components/assistant/AssistantDetails.tsx:98]: ✂️

**Review by flvndvd — COMMENTED** [review]
LGTM, just one comment to always display the try item.

**Pull request merged**